playbooks/installer/ospd/post.yml

```diff
@@ -30,6 +34,24 @@
                 name: "{{ item }}"
                 ansible_user: "heat-admin"
             with_items: "{{ groups['overcloud_nodes'] }}"
+
+          - name: add hosts to host list
+            become: no
+            add_host:
+                name="{{ item }}"
+                groups="{{ hostvars[item].provisioner.topology.nodes[item].groups | join(',') }}"
+                ansible_user="{{ user }}"
+                ansible_ssh_password="{{ password }}"
+                ansible_host="{{ hostvars[item].ansible_host }}"
+                ansible_ssh_private_key_file="{{ privkey }}"
+            with_items: "{{ groups['openstack_nodes'] }}"
```

Comment:
Is the undercloud key suitable for all `openstack_nodes`?
Is `heat-admin` user good for the undercloud?